exercises/matching-brackets/runtests.jl

```diff
@@ -0,0 +1,31 @@
+# canonical data version: 2.0.0
+
+using Test
+
+include("matching-brackets.jl")
+
+@testset "Determine if brackets in expression are correctly matched" begin
+    @test  matching_brackets("[]") 
+    @test !matching_brackets("[[")
+    @test !matching_brackets("}{")
+    @test !matching_brackets("{]")
+    @test  matching_brackets("{ }")
+end
+
+@testset "Determine if brackets in expression are correctly matched" begin
```

Comment:
Could you give these sets a unique name? Would be nicer and more useful to the student if the test summary shows what's being tested. Currently it's all the same:

```
[ Info: Testing matching-brackets
Test Summary:                                             | Pass  Total
Determine if brackets in expression are correctly matched |    5      5
Test Summary:                                             | Pass  Total
Determine if brackets in expression are correctly matched |    9      9
Test Summary:                                             | Pass  Total
Determine if brackets in expression are correctly matched |    3      3
```